Documentation/Testing/Tutorial/Acceptance.rst

```diff
@@ -0,0 +1,267 @@
+.. include:: /Includes.rst.txt
+.. index:: Testing; Project
+.. _testing-tutorial-acceptance:
+
+=======================================
+Acceptance testing of site_introduction
+=======================================
+
+Introduction
+============
+
+Testing entire projects is somehow different from Core and extension testing. As a developer
+or maintainer of a specific TYPO3 instance, you probably do not want to test extension details
+too much - those should have been tested on an extension level already. And you probably also
+do not want to check too many TYPO3 backend details but look for acceptance testing of your
+local development, stage and live frontend website instead.
+
+Project testing is thus probably wired into your specific CI and deployment environment. Maybe
+you want to automatically fire your acceptance tests as soon as some code has been merged to
+your projects develop branch and pushed to a staging system?
+
+Documenting all the different decisions that may have been taken by agencies and other project
+developers is way too much for this little document. We thus document only one example
+how project testing could work: We have some "site" repository based on `ddev
+<https://www.drud.com/what-is-ddev/>`_ and add basic acceptance testing to it, executed
+locally and by GitHub Actions.
+
+This is thought as an inspiration you may want to adapt for your project.
+
+
+Project site-introduction
+=========================
+
+The `site-introduction <https://github.com/TYPO3-Documentation/site-introduction>`_ TYPO3 project is a
+straight ddev based setup that aims to simplify handling the `introduction
+<https://github.com/FriendsOfTYPO3/introduction>`_ extension. It delivers everything needed
+to have a working introduction based project, to manage content and export it for new
+introduction extension releases.
+
+Since we're lazy and like well defined but simply working environments, this project is
+based on ddev. The repository is a simple project setup that defines a working
+TYPO3 instance. And we want to make sure we do not break main parts if we fiddle with it.
+Just like any other projects wants.
+
+The quick start for an own site based on this repository boils down to these commands, with
+more details mentioned in `README.md <https://github.com/TYPO3-Documentation/site-introduction/blob/main/README.md>`_:
+
+.. code-block:: shell
+
+    lolli@apoc /var/www/local $ git clone git@github.com:TYPO3-Documentation/site-introduction.git
+    lolli@apoc /var/www/local $ cd site-introduction
+    lolli@apoc /var/www/local/site-introduction $ ddev start
+    lolli@apoc /var/www/local/site-introduction $ ddev import-db --src=./data/db.sql
+    lolli@apoc /var/www/local/site-introduction $ ddev import-files --src=./assets
+    lolli@apoc /var/www/local/site-introduction $ ddev composer install
+
+This will start various containers: A database, a phpmyadmin instance, and a web server. If all
```

Comment:
phpadmin is no longer bundled with ddev - or at least not active per default ? (I'm not a ddev user on daily buisness. This should be rechecked here at least.

I know, in the past phpadmin container was stared for each project, but it seems to be no longer the case (ddev changes).

Reply:
I just moved this content unchanged, we will deal with it in a follow up